index.html

```diff
@@ -48,6 +48,194 @@ <h2 class="title">I’m a Software Developer</h2>
+        <div class="art-container1">
+          <h3 class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </h3>
+          <p class="profesional-Art-content">
+            A daily
+            selection of privately personalized reads;
+            no accounts or sign-ups required. has been the
+            industry's standard
+          </p>
+          <ul class="profesional-Art-ul">
+            <li class="profesional-Art-li">html</li>
+            <li class="profesional-Art-li">bootstrap</li>
+            <li class="profesional-Art-li">Ruby</li>
+          </ul>
+          <a href="#6" id="hide-me" class="profesional-Art-btm button">See Project</a>
+        </div>
+        <div class="art-container2">
+          <h3 class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </h3>
+          <p class="profesional-Art-content">
+            A daily
+            selection of privately personalized reads;
+            no accounts or sign-ups required. has been the
+            industry's standard
+          </p>
+          <ul class="profesional-Art-ul">
+            <li class="profesional-Art-li">html</li>
+            <li class="profesional-Art-li">bootstrap</li>
+            <li class="profesional-Art-li">Ruby</li>
+          </ul>
+          <a href="#7" id="hide-me" class="profesional-Art-btm button">See Project</a>
+        </div>
+      </section>
+      <section id="About-me" class="about">
+        <div class="about-me-footer">
+          <div class="detail">
+            <h2 class="about-me">About me</h2>
+            <p class="footer-text">
+              Hey there! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <a href="#" class="footer-btm button">Get My Resume</a>
+          </div>
+          <div class="image-bg"></div>
+          <div class="footer-img"></div>
+        </div>
+        <div class="footer-hr"> </div>
+        <div class="my-skills">
+          <div class="footer-list">
+            <h2 class="footer-heading">🔶 Language</h2>
+            <ul class="footer-ul">
+              <li class="footer-li">Javascript</li>
+              <li class="footer-li">Ruby</li>
+              <li class="footer-li">Html</li>
+              <li class="footer-li">Css</li>
+            </ul>
+          </div>
+          <div class="work">
+            <h2 class="footer-heading">🟧 Frameworks</h2>
+            <ul class="footer-ul">
+              <li class="footer-li">Bootstrap</li>
+              <li class="footer-li">Ruby on Rails</li>
+              <li class="footer-li">RSPec</li>
+              <li class="footer-li">CapyBara</li>
+              <li class="footer-li">Selenium</li>
+            </ul>
+          </div>
+          <div class="skill">
+            <h2 class="footer-heading">🟠 Skills</h2>
```

Comment:
- [ ] Please adhere to the `Figma` design as much as possible. I noticed that you are using the `icon` differently than provided by the *Figma* design template's section because following the requirements and implementing the correct features or UI helps to show how much you are *detail-oriented*, which makes you a better software developer. Kindly look at the below image.

<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-04-27 21-12-54](https://user-images.githubusercontent.com/66476329/234919757-7d429692-dfd9-4504-8e11-24a47d856969.png)


</td>
<td>

![Screenshot from 2023-04-27 21-16-12](https://user-images.githubusercontent.com/66476329/234919834-a1558cd5-b784-4435-8a0f-108119fbf066.png)


</td>
</tr>
</table>